### DISCUSSION TITLE
第6节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#150] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by yestolife** [comment]
[笔记+作业：OpenCompass大模型测评](http://t.csdnimg.cn/QyKBD)

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/zs1ytiyzfg4hc3kh?singleDoc# 《OpenCompass 大模型评测》

**Comment by DrZhang1208** [comment]
[OpenCompass 大模型评测](https://blog.csdn.net/qq_42866033/article/details/135757857?spm=1001.2014.3001.5501)

**Comment by xwhclaire** [comment]
[Lecture 6 OpenCompass 大模型评测](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/Lecture%206%20OpenCompass%20%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%AF%84%E6%B5%8B.md)

**Comment by Heitao5200** [comment]
https://gitee.com/heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK6.md

**Comment by tackhwa** [comment]
[第6课笔记+作业](https://nutritious-chill-7a9.notion.site/OpenCompass-2852fcfe5227459091d5d7bbcc6fec3d?pvs=4)

**Comment by ggfengxuan** [comment]
[6.OpenCompass 大模型评测笔记](https://v1y50l9n58a.feishu.cn/wiki/LtrpwekFpiEApkkeBWhcyUZ1nmh?from=from_copylink)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/ocn6btn9apd1ovta?singleDoc# 《lesson6 OpenCompass 大模型评测 笔记&作业》

**Comment by scorpionfree98** [comment]
[第六节课笔记+作业](http://t.csdnimg.cn/z3CIE)

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/6
